interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncountermanagerTable.php

```diff
@@ -339,7 +339,7 @@ public function getFileID($pid, $limit = 1)
         $query = "SELECT cc.id, pd.fname, pd.lname, pd.pid FROM ccda AS cc
 		    LEFT JOIN patient_data AS pd ON pd.pid=cc.pid
 		    WHERE cc.pid = ?
-		    ORDER BY cc.id DESC LIMIT $limit";
+		    ORDER BY cc.id DESC LIMIT " . escape_limit($limit);
```

Comment:
escaped above

Reply:
yep, double escaped :)
maybe should do following to really be sure :)
```
ORDER BY cc.id DESC LIMIT ";
for ($i=0; $i<1000000000; $i++) {
    $limit = escape_limit($limit);
}
$query .= $limit;
```

Reply:
I tend to try to set variables before using.